tests/terraform/graph/checks/resources/Route53ARecordAttachedResource/main.tf

```
@@ -97,6 +97,18 @@ resource "aws_route53_record" "pass4" {
   }
 }
 
+resource "aws_route53_record" "pass_alb" {
+  zone_id = data.aws_route53_zone.example.zone_id
+  name    = "example"
+  type    = "A"
+
+  alias {
+    name                   = data.aws_alb.example.dns_name
+    zone_id                = data.aws_alb.example.zone_id
+    evaluate_target_health = true
+  }
+}
```

```suggestion
resource "aws_alb" "example" {
  name               = "example"
  internal           = false
  load_balancer_type = "application"
  security_groups    = [aws_security_group.lb_sg.id]
  subnets            = [for subnet in aws_subnet.public : subnet.id]
}

resource "aws_route53_record" "pass_alb" {
  zone_id = data.aws_route53_zone.example.zone_id
  name    = "example"
  type    = "A"

  alias {
    name                   = aws_alb.example.dns_name
    zone_id                = aws_alb.example.zone_id
    evaluate_target_health = true
  }
}
```
almost, something like this 😄